content/concepts/basics/02-content-tree.md

```diff
@@ -4,38 +4,38 @@ metaTitle: "sensenet - Content Tree"
+Setting local only permissions that are not propagated to children is also possible within sensenet. Check the [document level permission article](/concepts/document-level-permissions) for more information.
 
 # Settings inheritance
 
-Settings are created for administrators or editors to let them customize the behavior of a certain feature. In sensenet these settings are stored as content in the content repository so you can take all the advantages of the tree structure in case of settings as well.
+Settings are created for administrators or editors to let them customize the behavior of a certain feature. In sensenet, these settings are stored as content in the content repository, allowing you to take advantage of the tree structure in the case of settings as well.
 
-Settings files can be global or local. Local settings files override global ones and they are applied only on the related part of the content repository. Every key in a settings file is overridable in another file with the same name under an appropriate position in the subtree. For example you can use a global document preview watermark setting for the whole repository, one for a project workspace and another one for a specific document library for contracts.
+Settings files can be global or local. Local settings files override global ones and are applied only on the related part of the content repository. Every key in a settings file is overridable in another file with the same name under an appropriate position in the subtree. For example, you can use a global document preview watermark setting for the whole repository: one for a project workspace and another one for a specific document library for contracts.
 
 # Content type inheritance
 
-Content types are defined in a type hierarchy: a content type may be inherited from another type automatically inheriting its fields and the underlying business logic (handler). A content type may only inherit fields from a single type because of the tree structure, multiple inheritance is not allowed. Inherited field configuration can be overridden in derived types (e.g. in a custom `CompanyUser` type inherited from the built-in User type will have all the fields that are defined on the `User` and can also have additional ones related to its specialities).
+Content types are defined in a type hierarchy: a content type may be inherited from another type, automatically inheriting its fields and the underlying business logic (handler). A content type may only inherit fields from a single type because of the tree structure. Multiple inheritance is not allowed. Inherited field configuration can be overridden in derived types. For example, a custom `CompanyUser` type inherited from the built-in User type will have all the fields that are defined on the `User`, and can also have additional ones related to its specialities).
```

Comment:
```suggestion
Content types are defined in a type hierarchy: a content type may be inherited from another type, automatically inheriting its fields and the underlying business logic (handler). A content type may only inherit fields from a single type because of the tree structure. Multiple inheritance is not allowed. Inherited field configuration can be overridden in derived types. For example, a custom `CompanyUser` type inherited from the built-in User type will have all the fields that are defined on the `User`, and can also have additional ones related to its specialities.
```